src/SUMMARY.md

```diff
@@ -58,6 +58,9 @@ Summary
 - [Constant Product AMM](./applications/constant-product-amm.md)
 - [TimeLock](./applications/timelock.md)
 - [Staking](./applications/staking.md)
+- [Crowdfunding](./applications/crowdfunding/crowdfunding.md)
+  - [Campaign Contract](./applications/crowdfunding/campaign.md)
+  - [CampaignFactory Contract](./applications/crowdfunding/factory.md)
```

Comment:
I think we could put all directly at top level, so the "Factory" word is directly visible

Reply:
I'm not sure I got you, did you mean something like:
- Crowdfunding Factory
  - Campaign Contract
  - CampaignFactory Contract
 
?

Reply:
Just:
```
- Crowdfunding Campaign
- AdvancedFactory: Crowdfunding
```
And you can start `AdvancedFactory: Crowdfunding` with something along the line of "We saw how to create a [Crowdfunding Campaign Contract], but managing multiple campaigns means deploying multiple contracts and tracking each one individually, which can be tiresome. In this example, we'll explore a smarter way to handle multiple deployments and keep track of each instance using the [Factory Pattern] with additional features for running multiple Crowdfunding campaings."...